### DISCUSSION TITLE
Obsidian&Zettelkasten&LYT构建第二大脑系列文章（持续更新）

### DISCUSSION THREAD

**Discussion by kingsun1980** [obsidianzh/forum#47] [实用技巧/最佳实践]
终于找到obsidian的中文社区啦！使用obsidian已经有几个月的时间，这个过程很受益，也踩过很多坑，我想把他们记录下来，既是一份成长记录，也是一次复盘思考。我将结合自己在工作、学习、交流过程中使用Obsidian构建第二大脑的具体实践过程，持续更新一系列文章，希望和愿意提升思维能力以及生产力的小伙伴，一起思考、实践、超越自我。以下是系列文章的总目录和链接，方便大家需要时阅读。**我的微信是：wunianwutang**，欢迎一起学习研讨，向各位报道，大伙多多关照！

- [构建属于自己的第二大脑](https://www.jianshu.com/p/d7bd0cb9d257)
- [Obsidian手把手教你快速上手（附核心配置）](https://www.jianshu.com/p/2def06958af6)
- [聊聊Zettelkasten如何构建第二大脑](https://www.jianshu.com/p/8cd19aacf7f0)
- [用Obsidian实现Zettelkasten看这一篇就够了（上）](https://www.jianshu.com/p/cd9388319366)
- [自己动手组建可远程实时同步的obsidian](https://www.jianshu.com/p/f3a200e4baa6)
- [Zettelkasten原子笔记与认知涌现之间还差了个LYT](https://www.jianshu.com/p/1e63bf2d8aa4)
- [MOC三阶段实操案例之“黄奇帆宏观经济思路MOC”构建全过程](https://www.jianshu.com/p/9f8d43924209)

文章持续更新中......

**Comment by makerjackie** [comment]
赞一个

**Reply by JimmyLv** [reply]
ya~ 有没有Obsidian微信讨论群呀？

**Comment by Quorafind** [comment]
很棒
